dts/arm64/rockchip/rockchip_rk3399.dtsi

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022 Huawei France Technologies SASU
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm64/armv8-a.dtsi>
+#include <zephyr/dt-bindings/interrupt-controller/arm-gic.h>
+#include <mem.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53";
+			reg = <0>;
+		};
+		cpu@1 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53";
+			reg = <1>;
+		};
+		cpu@2 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53";
+			reg = <2>;
+		};
+		cpu@3 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53";
+			reg = <3>;
+		};
+		cpu@4 {
```

Comment:
Just out of curiosity, generally, a53 and a72 belong to different cluster and thus have different `mpdir` aff formats, for example, a72's `mpdir` are something different like 0x100 0x200 or other formats? But here seems not the case (a72 and a53 are in the same cluster)?